__tests__/integration/select_query_binding.test.ts

```diff
@@ -268,4 +269,35 @@ describe('select with query binding', () => {
       )
     })
   })
+
+  describe('NULL parameter binding', () => {
+    const baseQuery: Pick<QueryParams, 'query' | 'format'> = {
+      query: 'SELECT number FROM numbers(3) WHERE {n:Nullable(String)} IS NULL',
+      format: 'CSV',
+    }
+
+    it('should work with nulls', async () => {
+      const rs = await client.query({
+        ...baseQuery,
+        query_params: {
+          n: null,
+        },
+      })
+
+      const response = await rs.text()
+      expect(response).toBe('0\n1\n2\n')
+    })
+
+    it('should with an explicit undefined', async () => {
```

Comment:
How does it behave with an undefined value?

Reply:
Missing key will fail and there is a test for that, cause the parameter is not bound at all. Explicit undefined and null values are sent as `\\N`